windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -85,6 +89,25 @@ namespace winrt::ReactNativeWebView::implementation {
                 WriteWebViewNavigationEventArg(webView, eventDataWriter);
                 eventDataWriter.WriteObjectEnd();
             });
+
+                
+        if (m_messagingEnabled) {
+            m_messageToken = webView.WebMessageReceived([this](auto const& sender, winrt::CoreWebView2WebMessageReceivedEventArgs const& args)
+                {
+                    try {
+                        auto message = args.TryGetWebMessageAsString();
+                        this->OnMessagePosted(message);
+                    }
+                    catch (std::exception e) {
```

Comment:
IIRC, exceptions from winrt are not std::exception (they're e.g. `winrt::hresult_error`). Since you aren't doing anything with the exception, you can just `catch (...) ` instead to catch any type

related: in general you should catch exceptions by `const&` so we don't need a copy ctor call